### PR TITLE
[travis] fix USN checks (Fixes #1622)

### LINKED ISSUE #1622: [CI] The job to check snaps for USNs fails
Example: https://api.travis-ci.com/v3/job/363506990/log.txt
```plain
- Mount snap "review-tools" (1720) (snap "review-tools" assumes unsupported features: command-chain (try to update snapd and refresh the core snap))
```

### PR DESCRIPTION
**Pull request by Saviq**
Also remove spurious stage.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,8 +34,6 @@ stages:
       AND ( branch = master
             OR branch =~ ^release/[\d\.]+$
             OR tag =~ ^v[\d\.]+$ )
-- name: process snaps
-  if: repo = MirServer/mir
```
Comment (AlanGriffiths) [inline .travis.yml:38]: Is this unrelated cleanup?

Comment (Saviq) [reply .travis.yml:38]: Yes, the same stage is configured at the top. [`git blame`](https://github.com/MirServer/mir/blame/c64dc50f6068ae2d0d2683867f1d2311b59da8eb/.travis.yml#L37L38) doesn't explain to me how this came to be here…

Comment (AlanGriffiths) [reply .travis.yml:38]: You have to look at the history.

- 09/07/18: I added it at the top
- 12/07/18: You added it at the bottom

As the duplication "worked" after merging nobody noticed before now.
